### PR TITLE
Add UI for creating just a channel.

### PR DESCRIPTION
**Pull request by jerith**
This can borrow a lot of its look from the first screen of the wizard.

### CODE REVIEW

**Comment by peterp** [comment]
:+1:

File: go/channel/templates/channel/new.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+</form>
+{% endblock %}
+
+
+{% block ondomready %}
+    {{ block.super }}
+    
+    // if a channel is already selected then we should show the
+    // configuration view for it.
+    $('input[name=channel]:checked').trigger('change')
```
Comment (hodgestar) [inline go/channel/templates/channel/new.html:52]: Missing semi-colon.

File: go/channel/templates/channel/new.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+    $('input[name=channel]').on('change', function() {
+        // hide both
+        $('.configuration').show()
```
Comment (hodgestar) [inline go/channel/templates/channel/new.html:56]: Missing semi-colon.

File: go/channel/templates/channel/new.html (hunk 1 above)
Comment (jerith) [inline go/channel/templates/channel/new.html:52]: I just copied this from an existing template.

File: go/channel/templates/channel/new.html (hunk 2 above)
Comment (jerith) [inline go/channel/templates/channel/new.html:56]: I just copied this from an existing template.

File: go/channel/templates/channel/new.html (hunk 1 above)
Comment (hodgestar) [inline go/channel/templates/channel/new.html:52]: Can we fix both templates then? It can be via a gist.

Comment (peterp) [inline go/channel/templates/channel/new.html:52]: I'll do it.

**Pull request merged**